spec/hbw/features/bp_form/translation_mock.yml

```diff
@@ -0,0 +1,211 @@
+## Structure of file:
+# method:
+#   url:
+#     body:
+#       response
+
+## NEW Structure of file:
+# <method>:
+#   <url>:
+#     - params: <body-1 or querystring-1>
```

Comment:
I have renamed "body" to params because this section is about:
"query string" (https://github.com/latera/homs/pull/586/files#diff-074616a70e9249c4ac4175e21aa10e27b76ebc44051d9adec842cd034c5fb3a8R19) 
and 
"body" (https://github.com/latera/homs/pull/586/files#diff-074616a70e9249c4ac4175e21aa10e27b76ebc44051d9adec842cd034c5fb3a8R167)

In alternative, we can name it "data"